jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/list/AdvancedSearchFiltersViewImplTest.java

```diff
@@ -102,4 +104,33 @@ public void testDateChange() {
         verify(toDate).setValue(null);
         verify(addCallback).accept(any(DateRange.class));
     }
+
+    @Test
+    public void testNumericInput() {
+        testValidKeyCode(KeyCodes.KEY_BACKSPACE);
+        testValidKeyCode(KeyCodes.KEY_NINE);
+        testValidKeyCode(KeyCodes.KEY_NUM_EIGHT);
+        testValidKeyCode(KeyCodes.KEY_NUM_ZERO);
+
+        testInvalidValidKeyCode(KeyCodes.KEY_NUM_MINUS);
+        testInvalidValidKeyCode(KeyCodes.KEY_NUM_PLUS);
+        testInvalidValidKeyCode(KeyCodes.KEY_SPACE);
+        testInvalidValidKeyCode(KeyCodes.KEY_INSERT);
+    }
+
+    protected void testInvalidValidKeyCode(int keyCode) {
```

Comment:
@bsremac that doesn't make sense :) fixing here: https://github.com/kiegroup/jbpm-wb/pull/738